infrastructure/ecs.tf

```diff
@@ -90,7 +90,8 @@ data "aws_iam_policy_document" "this" {
     effect = "Allow"
     actions = [
       "batch:DescribeJobQueues",
-      "batch:SubmitJob"
+      "batch:SubmitJob",
+      "batch:CreateEndpoint",
```

Comment:
sagemaker:CreateEndpoint

Reply:
Thanks @MotwaniM, I have updated the permission.